src/v1/reset_password.rs

```diff
@@ -0,0 +1,39 @@
+//! [POST /_synapse/admin/v1/reset_password/:user_id](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#reset-password)
+
+use ruma::api::ruma_api;
+use ruma::identifiers::UserId;
```

Comment:
Usually we do one `use` per crate.

```suggestion
use ruma::{api::ruma_api, identifiers::UserId};
```